configs/couplings/fesom-2.1-awiesm-2.1+echam-6.3.05p2-awiesm-2.1/fesom-2.1-awiesm-2.1+echam-6.3.05p2-awiesm-2.1.yaml

```diff
@@ -0,0 +1,8 @@
+components:
+- echam-6.3.05p2-awiesm-2.1
+- fesom-2.1-awiesm-2.1
+- oasis3mct-2.8-paleodyn
+coupling_changes:
+- sed -i '/set(FESOM_COUPLED/s/OFF/ON/g' fesom-2.1/CMakeLists.txt
+- sed -i '/ECHAM6_COUPLED/s/OFF/ON/g' echam-6.3.05p2/config/mh-linux
+- sed -ir '/..FC_DEFINE}__cpl_mpiom/s/..FC_DEFINE}__cpl_mpiom//g' echam-6.3.05p2/configure.ac
```

Comment:
The coupling to FESOM is handled with this `DFlags` switch, correct?

Reply:
Indeed, so the FESOM line can be removed.

Reply:
Do you mean the MPIOM line?

Reply:
I mean this line:
``` - sed -i '/set(FESOM_COUPLED/s/OFF/ON/g' fesom-2.1/CMakeLists.txt ```
It is not needed to modify the `CMakeLists.txt` if you have this already in the compilation command of FESOM:
https://github.com/esm-tools/esm_tools/blob/9c7cc77d9a5b0dad3a13525d144b1fc0a02215b3/configs/setups/awiesm/awiesm-2.1.yaml#L186-L190

https://github.com/esm-tools/esm_tools/blob/9c7cc77d9a5b0dad3a13525d144b1fc0a02215b3/configs/setups/awiesm/awiesm-2.1.yaml#L65-L79